pkgs/development/libraries/gdbm/default.nix

```diff
@@ -10,6 +10,8 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  configureFlags = "--enable-libgdbm-compat";
```

Comment:
This is actually a list of strings. Still works because nix is very forgiving, but I suggest to put `[ ]` around the flag.